### PR TITLE
Fix PIN button not visible when menu expands (#859)

### LINKED ISSUE #859: Fix PIN button not visible when menu expands
### Dev Home version

_No response_

### Windows build number

_No response_

### Other software

_No response_

### Steps to reproduce the bug

1. click add widget
2. expand all menu

### Expected result

_No response_

### Actual result

pin button disappear

### Included System Information

_No response_

### Included Extensions Information

_No response_

### PR DESCRIPTION
**Pull request by ssccinng**
## Summary of the pull request
Fixed the PIN button not visible issue
## References and relevant issues
#859 
## PR checklist
- [x] Closes #859 
- [ ] Tests added/passed
- [ ] Documentation updated

### CODE REVIEW

File: tools/Dashboard/DevHome.Dashboard/Views/AddWidgetDialog.xaml [hunk 1]
```diff
@@ -45,15 +45,16 @@
                         IsPaneToggleButtonVisible="False"
                         IsTitleBarAutoPaddingEnabled="False"
                         OpenPaneLength="218"
+                        MaxHeight="{StaticResource ContentDialogMaxHeight}"
```
Comment (betanzos) [inline tools/Dashboard/DevHome.Dashboard/Views/AddWidgetDialog.xaml:48]: The right value should be `654`

Comment (ssccinng) [reply tools/Dashboard/DevHome.Dashboard/Views/AddWidgetDialog.xaml:48]: Sorry I misunderstood the height☹️, thanks for correcting. But maybe 652 is better (?)

Comment (betanzos) [reply tools/Dashboard/DevHome.Dashboard/Views/AddWidgetDialog.xaml]: Could works too.

**Comment by EricJohnson327** [comment]
Closed and reopened to trigger new PR validation.

**Comment by EricJohnson327** [comment]
@ssccinng Please include before/after screenshots.

**Comment by krschau** [comment]
Closing, since this was superseded by the larger #1033

**Pull request closed without merging**